index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">  
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">  
+    <title>Survey Form</title>
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action=" hello" id="survey-form">
+        <p>Name</p>
+        <input type="text" placeholder="Enter your name" id="name">
+        <p>Email</p>
+        <input type="text" placeholder="Enter your Email" id="email">
+        <p>Age</p>
+        <input type="number" placeholder="Enter your age" id="number" min="16" max="100">
+        <p>Which option best describes your current role?</p>
+        <select name="name1" id="select1"><option label="label1" value="Student">Student</option><option value="Full time job">Full time job</option></select>
+
+
+    </form>
+    
```

Comment:
It's considered the best practices include the _main_ tag, and put it inside the selected code. As show below:

<main>
    <form>
    </form>
</main>

Try to do this change and your code will be cleaner! 😉

Reply:
Good observation and communication; As I have mentioned above this is related to sematic HTML and it should be a required change.

Reply:
Thanks for your words!